pkg/strings/strings.go

```diff
@@ -0,0 +1,36 @@
+package strings
+
+import (
+	"bufio"
+	"strings"
+)
+
+func Contains(s []string, e string) bool {
```

Comment:
`s/s/strings/`, `s/e/expected/` or `compare`. other functions use more decriptive variables, like `input`.

Reply:
I used needle/haystack, I can change again if this is harder to read than it should.